docs/book/src/SUMMARY.md

```diff
@@ -11,10 +11,12 @@
     - [Kubeadm based bootstrap](./tasks/kubeadm-bootstrap.md)
     - [Upgrading management and workload clusters](./tasks/upgrading-clusters.md)
     - [Upgrading Cluster API components](./tasks/upgrading-cluster-api-versions.md)
-    - [Configure a MachineHealthCheck](./tasks/healthcheck.md)
     - [Kubeadm based control plane management](./tasks/kubeadm-control-plane.md)
     - [Updating Machine Infrastructure and Bootstrap Templates](tasks/updating-machine-templates.md)
-    - [Using the Cluster Autoscaler](./tasks/cluster-autoscaler.md)
+    - [Automated worker Machine management](./tasks/automated-worker-machine-management/index.md)
+      - [Scaling](./tasks/automated-worker-machine-management/scaling.md)
+      - [Auto scaling](./tasks/automated-worker-machine-management/auto-scaling.md)
+      - [Health checking](./tasks/automated-worker-machine-management/healthchecking.md)
```

Comment:
Health checking is both for control plane and worker nodes...
What about calling this new section "Automated machine management" and explicitly calling out when a paragraph applies only to a subset of machines